packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -843,4 +843,44 @@ export interface CallCompositeStrings {
    * Error message when the meeting identifier or passcode is invalid
    */
   invalidMeetingIdentifier: string;
+  /* @conditional-compile-remove(teams-meeting-conference) */
+  /**
+   * Error message when the meeting identifier or passcode is invalid
+   */
+  phoneCallMoreButtonLabel: string;
```

Comment:
If we are keeping these strings in the Composite but not in the components (since we aren't exporting them) we might want to make the composite own the component. We should talk to Alex P and Tay if we want to export the component.

Reply:
I agree this, looks odd that the strings are in composites but only being using in a file in the components package

Reply:
Thanks, i've moved strings into component project